File: hackerrank/problem solving/algorithms/1 - warmup/2 - simple array sum/simple_array_sum.rb
# https://www.hackerrank.com/challenges/simple-array-sum/problem?isFullScreen=true

def main
  gets
  puts sum_stdin
end

# n: number of integers in the input line
# T: O(n)
# S: O(n) extra space
def sum_stdin
  gets.split.map!(&:to_i).sum
end

main if __FILE__ == $0

File: hackerrank/problem solving/algorithms/1 - warmup/2 - simple array sum/simple_array_sum.py
# https://www.hackerrank.com/challenges/simple-array-sum/problem?isFullScreen=true

def main():
    input()
    print(sum_stdin())


# n: number of integers in the input line
# T: O(n)
# S: O(n) extra space
def sum_stdin():
    return sum(map(int, input().split()))


if __name__ == '__main__':
    main()
